src/components/carousel/index.tsx

```diff
@@ -41,23 +44,10 @@ const Carousel: FC<ICarousel> = ({ children }) => {
         })}
       </div>
 
-      <div style={styles.controlContainer}>
-        <button
-          style={styles.iconStart}
-          onClick={leftClickHandler}
-          disabled={!hasPrevSlide}
-        >
-          &lt;
-        </button>
-
-        <button
-          style={styles.iconEnd}
-          onClick={rightClickHandler}
-          disabled={!hasNextSlide}
-        >
-          &gt;
-        </button>
-      </div>
+      <CarouselControl >
+        <Button onClick={leftClickHandler} disabled={!hasPrevSlide} ><LeftIcon /></Button>
+        <Button onClick={rightClickHandler} disabled={!hasNextSlide} ><RightIcon /></Button>
+      </CarouselControl>
```

Comment:
```suggestion
      <CarouselControl variant="arrows" position="bottom" customStyles={props.controlStyles}/>
```
Later we will have dynamic change for the control varian and position.